Source/Controllers/MainViewControllers/ConnectionView/SPConnectionController.m

```diff
@@ -3766,11 +3685,10 @@ - (void)dealloc
 	[self removeObserver:self forKeyPath:SPFavoriteSSLCertificateFileLocationEnabledKey];
 	[self removeObserver:self forKeyPath:SPFavoriteSSLCertificateFileLocationKey];
 	[self removeObserver:self forKeyPath:SPFavoriteSSLCACertFileLocationEnabledKey];
-	[self removeObserver:self forKeyPath:SPFavoriteSSLCACertFileLocationKey];
+    [self removeObserver:self forKeyPath:SPFavoriteSSLCACertFileLocationKey];
+    [self removeObserver:self forKeyPath:SPBookmarksChangedNotification];
```

Comment:
Looks like we got some padding inconsistencies here

Reply:
fixed